docs/src/components/MenuSelect.md

```diff
@@ -35,7 +34,8 @@ class-names | Object | `{}` | Override class names | no
 Name | Scope | Description
 ---|---|---
 default | `{ items: Array<{label: String, value: String}>, canRefine: Boolean, refine: String => void, createURL: () => String }` | Slot to override the DOM output
-item | `{ item: {label: String, value: String} }` | Slot to override the content of the `<option>`. Make sure to use a `<template>` tag here, since anything else is invalid DOM.
+item | `{ item: {label: String, value: String} }` | Slot to override the content of the `<option>`. Make sure to use a `<template>` tag as root here, since anything else is invalid DOM.
+all | `{}` | Label for the "no currently selected" optionMake sure to use a `<template>` tag as root here, since anything else is invalid DOM.
```

Comment:
```suggestion
 defaultOption | `{}` | Label for the "no currently selected" optionMake sure to use a `<template>` tag as root here, since anything else is invalid DOM.
```